install.sh

```diff
@@ -39,10 +39,16 @@ fi
 
 # Download latest alohacam binary.
 if [ -z "$SKIP_DOWNLOAD" ]; then
-    curl -o alohacam $DOWNLOAD_URL/release/$VERSION/alohacam-$PLAT-$ARCH
+    curl -L -o alohacam $DOWNLOAD_URL/release/$VERSION/alohacam-$PLAT-$ARCH
     chmod a+x alohacam
 fi
 
+# Install rng-tools to ensure have sufficient entropy for certificates
+if [ ! -e /usr/sbin/rngd ]; then
+    echo "Installing rng-tools to ensure sufficient entropy for certificates"
```

Comment:
It feels like this belongs further down in the script (near the camera driver section). Also, do you want to add `(requires sudo)` to remain consistent with other steps?

Reply:
Good point on requires sudo. I think it should be before the OpenSSL key generation though as that also requires entropy.

Reply:
Ah, right, good point.